### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,23 +51,23 @@
     "@babel/plugin-transform-runtime": "^7.4.4",
     "@babel/runtime": "^7.4.4",
     "@kiwicom/eslint-config": "^4.0.1",
-    "@kiwicom/monorepo": "^0.19.0",
+    "@kiwicom/monorepo": "^0.21.0",
```
Comment (mrtnzlml) [inline package.json]:
```diff
-    "@kiwicom/monorepo": "^0.21.0",
+    "@kiwicom/monorepo-utils": "^0.21.0",
```

Comment (tbergq) [reply package.json]: done in #1623 

Same package with new name, or are the also any differences @mrtnzlml

Comment (mrtnzlml) [reply package.json]: There shouldn't be any significant changes, only new name.

Comment (tbergq) [reply package.json]: Ok, thanks, New package has already been added 😊

**Pull request merged**